.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug Report
+about: Create a report to help us improve GNU Radio
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# 1. Have you looked in our documentation and asked the community?
+
+<!--- [Before you open an issue we suggest checking the following: -->
+<!--- - [The wiki](https://wiki.gnuradio.org/index.php/Main_Page) -->
+<!--- - [The GNU Radio Blocks documentation](https://wiki.gnuradio.org/index.php/Category:Block_Docs) -->
+<!--- - [The User manual](https://wiki.gnuradio.org/index.php/Usage_Manual) -->
+<!--- - [GNU Radio community](https://wiki.gnuradio.org/index.php/Chat) -->
+<!--- - or on [the mailing list](https://wiki.gnuradio.org/index.php/MailingLists)] -->
+
+# 2. GNU Radio version
```

Comment:
IMO this should be switched to "System Information" to include:
- GR version ( gnuradio-config-info --version)
- GR installation method (and remove "# 3. How did you install GNU Radio" )
- OS / Kernel version ( cat /etc/lsb-release, uname -a)
- VOLK version (volk-config-info --version )
-

Reply:
I think the things we need to know are usually:

* OS
   - [ ] Linux
   - [ ] Windows 8 or later
   - [ ] MacOS
   - [ ] Other
* Installation method:
  * native packages for
    - [ ] Debian (version: ____ )
    - [ ] Ubuntu (version: ____ )
    - [ ] Fedora (version: ____ )
    - [ ] macports
    - [ ] anaconda
    - [ ] Other
   - [ ] GNU Radio's official PPA repos (Ubuntu version: ____ )
   - [ ] PyBOMBS
   - [ ] Other
 
VOLK, kernel versions are rarely necessary to know, if that becomes relevant, we're probably already in a discussion.